### PR TITLE
Support for multiple Qute template root

### PR DESCRIPTION
**Pull request by angelozerr**
Support for multiple Qute template root

### CODE REVIEW

File: src/qute/languageServer/client.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { LanguageClient } from 'vscode-languageclient/node';
 import { JavaExtensionAPI } from '../../extension';
 import { sendTelemetry } from '../../utils/telemetryUtils';
 import { QuteClientCommandConstants } from '../commands/commandConstants';
-import { registerQuteLSDependentCommands, registerVSCodeQuteCommands, synchronizeQuteValidationButton } from '../commands/registerCommands';
+import { registerQuteLSDependentCommands, registerUpdateQuteLanguageIdCommand, registerVSCodeQuteCommands, synchronizeQuteValidationButton } from '../commands/registerCommands';
```
Comment (datho7561) [inline src/qute/languageServer/client.ts]: VS Code is telling me that `registerUpdateQuteLanguageIdCommand` doesn't exist, I don't see it in `src/qute/commands/registerCommands.ts`

Comment (angelozerr) [reply src/qute/languageServer/client.ts]: My bad, I forget to push teh client.ts which remove this command. Sorry @datho7561  -(

**Comment by datho7561** [comment]
@angelozerr is this PR ready? I think it looks good and works from my testing

**Review by datho7561 — APPROVED** [review]
Looks good. Thanks Angelo!

**Comment by angelozerr** [comment]
Yes the PR is ready. I will merge it when the PR from quarkus ls will be merged.

**Pull request merged**